<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetWidgetValue</class>
 <widget class="QMainWindow" name="SetWidgetValue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SetWidgetValue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpConnection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QPushButton" name="btnConnect">
     <property name="geometry">
      <rect>
       <x>228</x>
       <y>50</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDisconnect">
     <property name="geometry">
      <rect>
       <x>228</x>
       <y>90</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="grpPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Port</string>
     </property>
     <widget class="QRadioButton" name="rdoTTYUSB0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ttyUSB0</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdoTTYMXC7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ttymxc7</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edPort">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdoComPort">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>119</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>COM</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpCommand">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Command</string>
    </property>
    <widget class="QPushButton" name="btnSetValue_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Set value 0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSetValue_50">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Set value 50</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSetValue_100">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Set value 100</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbReplyMessage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbSentMessage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="sbValue">
    <property name="geometry">
     <rect>
      <x>457</x>
      <y>46</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>41</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edWidgetID">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>41</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbValue">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
